e2e/internal/client/aws.go

```diff
@@ -0,0 +1,162 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	cmd "github.com/aws/copilot-cli/e2e/internal/command"
+)
+
+// AWS is a wrapper around aws commands.
+type AWS struct{}
+
+// VPCStackOutput is the output for VPC stack.
+type VPCStackOutput struct {
+	OutputKey   string
+	OutputValue string
+	ExportName  string
+}
+
+// NewAWS returns a wrapper around AWS commands.
+func NewAWS() *AWS {
+	return &AWS{}
+}
+
+/*CreateStack runs:
+aws cloudformation create-stack
+	--stack-name $name
+	--template-body $templatePath
+*/
+func (a *AWS) CreateStack(name, templatePath string) error {
+	command := strings.Join([]string{
+		"cloudformation",
+		"create-stack",
+		"--stack-name", name,
+		"--template-body", templatePath,
+	}, " ")
+	return a.exec(command)
+}
+
+/*StackCreateComplete runs:
+aws cloudformation wait stack-create-complete
+	--stack-name $name
+*/
+func (a *AWS) StackCreateComplete(name string) error {
```

Comment:
Nit: can this be renamed to WaitStackCreateComplete? I was not sure when I read it in context if it called `wait stack-create-complete` or if it was a one-off check for completion.

Reply:
good idea. will fix then